### DISCUSSION TITLE
Where is stack usage support?

### DISCUSSION THREAD

**Discussion by nordlow** [ccache/ccache#159] [Support]
https://github.com/ccache/ccache/pull/135#issuecomment-281154501

Support for `-fstack-usage` is on master but not on 3.3.4. Why?

**Comment by afbjorklund** [comment]
Probably hasn't been backported to the maint release ?

**Comment by nordlow** [comment]
Could you please do that? Or is it scheduled for 3.4?

**Comment by jrosdahl** [comment]
See comment in #135.

**Comment by nordlow** [comment]
Thanks.
